app/src/main/java/com/sdpteam/connectout/client_validation/EventCreationValidationUtils.java

```diff
@@ -0,0 +1,36 @@
+package com.sdpteam.connectout.client_validation;
+
+import android.widget.EditText;
+import android.widget.RadioButton;
+
+public class EventCreationValidationUtils {
+
```

Comment:
I would advise to rename this class `EventValidator`, as it makes it more clear.

Reply:
does EventCreationValidator works for you ?